### ISSUE TITLE
[Feature request} Playback speed

### ISSUE THREAD

**Issue by robcazzaro** [mopidy/mopidy#1667]
I'm using Mopidy with podcast extensions, and works really well. I miss one of the features in my podcast app on Android, though: the ability to speed up the playback, and change playback speed to an higher value (usually 1.4x). The majority of podcasts are perfectly intelligible at higher than normal playback speed, and it helps maximizing the number of podcasts I can listen to

Would it be possible to add a playback speed option?

**Comment by pataquets** [comment]
Same issue, same use case.
@jodal: what would entail implementing this?
On a cursory read, I've seen no playback on the Podcast, plugin. Seems to me that it only provides the URL to Mopidy and it's up to Mopidy to stream & play. disclaimer: I'm no Python expert.
I guess it would be somewhere in core, maybe `./mopidy/audio/actor.py`? Also, setting the speed would maybe be a matter of looking for some GST property? (not familiar with it)

**Comment by kingosticks** [comment]
> On a cursory read, I've seen no playback on the Podcast, plugin. Seems to me that it only provides the URL to Mopidy and it's up to Mopidy to stream & play

Yes. Pretty much all extensions delegate everything playback related to Mopidy's normal gstreamer pipeline in the file you mentioned. 

There's an example at https://gstreamer.freedesktop.org/documentation/tutorials/basic/playback-speed.html on (one way of?) doing this. The GStreamer part looks fairly simple and we'd just need to decide how to expose it. I don't think mpd has this feature so there's total freedom there.

**Comment by Klotzi111** [comment]
It would be great if the playback speed would be associated with the item to play. So that I can add the same song twice but with different playback speeds. And other songs still play at normal speed.
The queue and playlists would need to be able to store the playback speed with each item for that.
